department-of-veterans-affairs/caseflow
# Review assignments page

Connects #5821 

### Description
This PR adds a page for reviewing the judge and RO assignments. I will add the table of assignments in a future PR.

### Testing Plan
1. Go to /hearings/schedule/build/upload
1. Fill out the form and hit 'Continue'
1. Confirm the next page renders correctly
